src/frontend/planner_test/tests/testdata/batch_dist_agg.yaml

```diff
@@ -15,8 +15,8 @@
     └─BatchSimpleAgg { aggs: [max(s.v)] }
       └─BatchScan { table: s, columns: [s.v], distribution: Single }
   batch_local_plan: |
-    BatchExchange { order: [], dist: Single }
-    └─BatchSimpleAgg { aggs: [max(s.v)] }
+    BatchSimpleAgg { aggs: [max(s.v)] }
+    └─BatchExchange { order: [], dist: Single }
```

Comment:
Not sure about `Agg`. Should we push this to `ComputeNode` as well? Which operators should compute node run / frontend run.

Seems like there's only a requirement that compute node runs table scan.

For the rest, which should also run on compute node? For now I'm just making educated guesses: E.g. `BatchTopN` and `BatchFilter` should run on compute node, if they are above `BatchScan`, to reduce data sent over RPC.

Seems like `BatchSimpleAgg` should also be on `ComputeNode`? But I'm not very sure.

Perhaps it requires benchmarks to know for sure?